### DISCUSSION TITLE
Add support for outputLatency with audio worklets

### DISCUSSION THREAD

**Discussion by FloRecisio** [emscripten-core/emscripten#22950] [Ideas]
Hello! I was wondering if it would be possible to access audio latency via emscripten, I guess so? 

https://developer.mozilla.org/en-US/docs/Web/API/AudioContext/outputLatency

I might take a look at it one day, if it's not too complicated to implement.

**Comment by cwoffenden** [comment]
It's part of the changes I'd like to make (conversation started in #22949). I'd like to add a filled struct with number of samples, sample rate and latency (and then anything else as the API grows).
